src/Simulation.h

```diff
@@ -450,6 +456,12 @@ class Simulation {
 	/** @brief Refresh particle IDs to continuous numbering*/
 	void refreshParticleIDs();
 
+	/** @brief Checks if Simsteps or MaxWallTime are reached */
+	bool keepRunning();
+
+	double _maxWallTime = -1;
+	bool _wallTimeEnabled = false;
```

Comment:
these two values should be private, not public

Reply:
3f1cca8